modules/ROOT/pages/basic-authentication-simple-concept.adoc

```diff
@@ -5,29 +5,34 @@ endif::[]
 
 [width="100%", cols="5,15"]
 |===
->s| Policy Name | Basic Authentication: Simple
+>s| Policy Name | Basic Authentication - Simple
```

Comment:
Is the hyphen actually part of the name?

Reply:
yup! Done! Thanks!

Reply:
Actually not. so changed it back.